tests/test_erosita.py

```diff
@@ -0,0 +1,875 @@
+        self.assertTrue(os.path.exists('test_data/erosita_calpv_raw/300010/300010.txt'))
+        self.assertFalse(os.path.exists('test_data/erosita_calpv_raw/temp_download'))
+
+    def test_directory_formatting_files_in_three_folders(self):
+        # writing some files to test the function with
+        # for testing purposes I am changing this attribute so the test files get written to the test_data folder
+        self.filtered._top_level_output_path = 'test_data/'
+        # self.raw_data_path is now test_data/erosita_calpv_raw
+
+        # setting up my fake downloaded data
+        # defining my path for fake downloaded data to go in
+        path = 'test_data/erosita_calpv_raw/temp_download/EFEDS/EFEDS/EFEDS/'
+        # making the directories
+        os.makedirs(os.path.dirname(path), exist_ok=True)
+        # making the files
+        obs_ids = ['300007', '300008', '300009', '300010']
+        for obs in obs_ids:
+            with open(path + obs + '.txt', 'w') as f:
+                f.write('testing')
+        # when downloading calpv data they all come with extra pdfs
+            with open(path + obs + 'eRO' + '.txt', 'w') as f:
+                f.write('testing')
+        
+        self.filtered._directory_formatting()
+
+        self.assertTrue(os.path.exists('test_data/erosita_calpv_raw/300007/300007.txt'))
+        self.assertTrue(os.path.exists('test_data/erosita_calpv_raw/300008/300008.txt'))
+        self.assertTrue(os.path.exists('test_data/erosita_calpv_raw/300009/300009.txt'))
+        self.assertTrue(os.path.exists('test_data/erosita_calpv_raw/300010/300010.txt'))
+        self.assertFalse(os.path.exists('test_data/erosita_calpv_raw/temp_download'))
+    
+    def test_get_evlist_path_from_obs(self):
+        self.defaults._top_level_output_path = 'test_data/'
+        with patch('daxa.mission.erosita.os.listdir') as mock_listdir:
+            mock_listdir.return_value = ['fm00_300004_020_EventList_c001.fits',
+                                         'fm00_300004_020_EventList_c001_if123.fits']
+            
+            result = self.defaults.get_evlist_path_from_obs('300004')
+
+        self.assertEqual(result, 'test_data/erosita_calpv_raw/300004/fm00_300004_020_EventList_c001.fits')
+    
+class TesteROSITACalPV_download(unittest.TestCase):
+    '''
+    Putting this test into a separate class since it needs a lot of patches
+    '''
+    def setUp(self):
+        self.etacha = eROSITACalPV(fields='eta cha')
+        self.etacha_insts = eROSITACalPV(fields='eta cha', insts=['TM1', 'TM2'])
+        self.survey = eROSITACalPV(fields='survey')
+
+        self.mock_dir_frmt = patch.object(eROSITACalPV, '_directory_formatting').start()
+        self.mock_inst_filt = patch.object(eROSITACalPV, '_inst_filtering').start()
+        self.mock_down_call = patch.object(eROSITACalPV, '_download_call').start()
+        self.mock_get_evlist = patch.object(eROSITACalPV, 'get_evlist_path_from_obs').start()
```

Comment:
As per my higher up comment, this method name has changed, but it should function the same